src/components/Search/SearchResultPanel.js

```diff
@@ -14,6 +20,9 @@ const Card = ({ title, author, children }) => (
         {children}
       </FlexView>
     </FlexView>
+    <FlexView shrink basis='100%' vAlignContent='center' hAlignContent='right'>
+      <Button label='More details' onClick={() => {}} {...buttonProps} />
+    </FlexView>
```

Comment:
`grow` instaed of `shrink basis='100%'` should be fine (replaceing it also in the sibling above (`className='description')